static/js/simulator.js

```diff
@@ -1361,6 +1364,104 @@ function LDQ(computer) {
     computer.mq_register.update_contents(computer.storage_register);
 }
 
+/**
+ * Emulates the IBM 704 Store MQ (STQ) operation.
+ *
+ * Stores the value of the Multiplier-Quotient Register into General Memory at the specified address.
+ *
+ * @param {IBM_704} computer    Machine to execute instruction on.
+ * @param {number}  address     Address of word to be changed.
+ */
+function STQ(computer, address) {
+    computer.general_memory[address].update_contents(computer.mq_register);
+}
+
+/**
+ * Emulates the IBM 704 Multiply (MPY) operation.
+ *
+ * Multiplies the fixed-point value of the storage register by the fixed-point value of the Multiplier-
+ * Quotient Register.  The 35 most significant bits are stored in the Accumulator, and the 35 least
+ * significant bits are placed in the Multiplier-Quotient Register.
+ *
+ * @param {IBM_704} computer    Machine to execute instruction on.
+ */
+function MPY(computer) {
+    let result = computer.storage_register.fixed_point*computer.mq_register.fixed_point;
```

Comment:
`const` and please always space infix operators (around `*` here) for readability